icechunk/src/session.rs

```diff
@@ -1708,6 +1899,7 @@ async fn fetch_manifest(
 /// available in `from` and `to` arguments.
 ///
 /// Yes, this is horrible.
+#[allow(dead_code)]
```

Comment:
we no longer use this shit?

Reply:
Yeah, i do a pass to group the references in to a manifest shard, so I just accumulate in that pass. I can delete.